acme.go

```diff
@@ -330,7 +330,7 @@ func (c *Client) Authorize(ctx context.Context, domain string) (*Authorization,
 	if err := json.NewDecoder(res.Body).Decode(&v); err != nil {
 		return nil, fmt.Errorf("acme: invalid response: %v", err)
 	}
-	if v.Status != StatusPending {
+	if v.Status != StatusPending && v.Status != StatusValid {
```

Comment:
@titanous I've just realized this changes the acme package and so needs to be in the x/crypto/acme, too. Otherwise, it'll be overwritten next time I sync.

Do you mind creating a CL at go-review.googlesource.com? You can add me (ddos@google) as a reviewer if you want, but I'll ask for a test :)

If you don't have time, no worries, I can do it. Just let me know.